sdv/tabular/copulas.py

```diff
@@ -318,6 +318,15 @@ def _sample(self, num_rows, conditions=None):
         """
         return self._model.sample(num_rows, conditions=conditions)
 
+    def _set_random_state(self, random_state):
+        """Set the random state of the model's random number generator.
+
+        Args:
+            random_state (int, np.random.RandomState, or None):
```

Comment:
What's the expected behavior if it's passed None? Maybe add a test for it, if the behavior is any different than passing an integer.

Reply:
It should set the underlying model's random state to None. Not sure if we should test it here since it's the internal model behavior (e.g. within CTGAN and copulas)